### PR TITLE
Updated Zipkin spec for remoteEndpoint mapping.

### PR DESCRIPTION
**Pull request by CodeBlanch**
See open-telemetry/opentelemetry-dotnet#483.

Recreated PR (old one was #474) to fix CLA issue.

### CODE REVIEW

File: specification/exporter-zipkin.md [hunk 1]
```diff
@@ -64,6 +63,22 @@ Zipkin.
...
+|---|---|---|
+|1|net.peer.name|[OpenTelemetry adopted attribute for remote hostname, or similar.](./data-span-general.md#general-network-connection-attributes)|
+|1|peer.service|Remote service name defined in OpenTracing specification.|
+|2|peer.hostname|Remote hostname defined in OpenTracing specification.|
+|2|peer.address|Remote address defined in OpenTracing specification.|
+|2|net.peer.ip & net.peer.port|[OpenTelemetry adopted attribute for remote address of the peer.](./data-span-general.md#general-network-connection-attributes)|
+|3|http.host|Commonly used address attribute for Http Spans.|
+|3|db.instance|Commonly used address attribute for DB Spans.|
+
+Lowest priority match should be selected. In the event that multiple hits occur at the same priority level (Ex: `net.peer.name` & `peer.service` are both specified), take the value from the first attribute matched. `net.peer.ip` can be used by itself as remote endpoint but should be combined with `net.peer.port` if it is also present.
```
Comment (Oberon00) [inline specification/exporter-zipkin.md]: "Lowest priority match should be selected." So the lowest priority match has the highest priority? 😕

Comment (CodeBlanch) [reply specification/exporter-zipkin.md]: Sigh :) For whatever reason, it doesn't bug me. Think about a priority 1 issue vs a priority 5? Sometimes lower is more urgent. But I'm happy to change it to reduce confusion. How about "rank" instead?

File: specification/exporter-zipkin.md [hunk 2]
```diff
@@ -64,6 +63,22 @@ Zipkin.
...
 |`SpanKind.INTERNAL`|`null` |must be omitted (set to `null`)|
 
+### Remote endpoint
+
+If Zipkin `SpanKind` resolves to either `SpanKind.CLIENT` or `SpanKind.PRODUCER` then the service SHOULD specify remote endpoint otherwise Zipkin won't treat the Span as a dependency. OpenTelemetry doesn't have a perfectly expressive field for the destination service, so the following mapping is applied:
+
+|Priority|Attribute Name|Reason|
+|---|---|---|
+|1|net.peer.name|[OpenTelemetry adopted attribute for remote hostname, or similar.](./data-span-general.md#general-network-connection-attributes)|
+|1|peer.service|Remote service name defined in OpenTracing specification.|
```
Comment (anuraaga) [inline specification/exporter-zipkin.md:73]: `peer.service` was added to OpenTelemetry with this use case in mind, I think we can use it as the highest priority.

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/span-general.md#general-remote-service-attributes

Comment (anuraaga) [reply specification/exporter-zipkin.md:73]: Oh actually I noticed some values have same priority - what's the decision mechanism within a priority?

Comment (CodeBlanch) [reply specification/exporter-zipkin.md]: @anuraaga There was a note in there below the table about what to do in that case, but I cleaned it up and simplified things for #776. I put `peer.service` at the top. We have been using that in opentelemetry-dotnet for a while, just didn't update this doc 😄

**Pull request closed without merging**